cmd/makaroni/main.go

```diff
@@ -16,7 +17,15 @@ func main() {
 		log.Fatalln(err)
 	}
 	multipartMaxMemory := flag.Int64("multipart-max-memory", multipartMaxMemoryEnv, "Maximum memory for multipart form parser")
-	resultURLPrefix := flag.String("result-url-prefix", os.Getenv("MKRN_RESULT_URL_PREFIX"), "Upload result URL prefix.")
+	domain := flag.String("domain-url", os.Getenv("MKRN_DOMAIN_URL"), "Domain url with schema.")
```

Comment:
It's not a domain URL. The app could be deployed on any path. I think it's wiser to call it like `INDEX_URL`.